release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -2741,6 +2741,9 @@ The following code provides an example of the policy updates with both names bei
 
 ----
 
+* Resetting a MAC address on an SR-IOV virtual function (VF) upon pod deletion may fail for Intel E810 NICs.
```

Comment:
```suggestion
* Resetting a MAC address on an SR-IOV virtual function (VF) upon pod deletion might fail for Intel E810 NICs.
```

From [IBM Style Word Usage](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#m): "Use _may_ only to indicate permission. Otherwise, use _can_ to indicate ability, or use _might_ to indicate possibility."